### PR TITLE
Added Twitter handle column in sponsor model. #309

### LINKED ISSUE #309: Sponsor model should support Twitter handles
Most sponsor companies have Twitter accounts. When we tweet about sponsorship (related: #244), it would be nice to at-mention the company. Adding a column to the Sponsor model makes it easier to work with this data in an automated / scheduled fashion.

Steps:
1. Add a column to the Sponsor model
2. Generate an alembic migration

### PR DESCRIPTION
**Pull request by RaghavPro**
Closes #309

### CODE REVIEW

File: pygotham/sponsors/models.py [hunk 1]
```diff
@@ -56,6 +56,7 @@ class Sponsor(db.Model):
     contact_email = db.Column(db.String(255))
     accepted = db.Column(db.Boolean)
     payment_received = db.Column(db.Boolean)
+    twitter_handle = db.Column(db.String(255))
```
Comment (dirn) [inline pygotham/sponsors/models.py]: We should use the same type and constraints [used elsewhere](https://github.com/PyGotham/pygotham/blob/90193277c175eecfd49e9514e971484d035d3526/pygotham/users/models.py#L67).

Comment (RaghavPro) [inline pygotham/sponsors/models.py]: Ah. Updated.
Small change. Don't want to add myself in authors file.

**Comment by dirn** [comment]
We're investigating the changes to pydocstyle that cause the check to fail. Once we figure out how we want to proceed, we'll push up a commit you can rebase from.

**Comment by dirn** [comment]
@RaghavPro builds are once again passing. Please rebase your branch so we can get this merged.

**Comment by RaghavPro** [comment]
Updated.

**Comment by dirn** [comment]
Thank you for your contribution.

**Pull request merged**